static/js/lezer-parsers/language-packages.ts

```diff
@@ -0,0 +1,241 @@
+import { parser as level1Parser} from './level1-parser'
+import { parser as level2Parser} from './level2-parser'
+import { parser as level3Parser} from './level3-parser'
+import { parser as level4Parser} from './level4-parser'
+import { parser as level5Parser} from './level5-parser'
+import { parser as level6Parser} from './level6-parser'
+import { parser as level7Parser } from './level7-parser'
+import { parser as level8Parser } from './level8-parser'
+import { parser as level10Parser } from './level10-parser'
+import { parser as level11Parser } from './level11-parser'
+import { parser as level12Parser } from './level12-parser'
+import { parser as level13Parser } from './level13-parser'
+import { parser as level14Parser } from './level14-parser'
+import { parser as level15Parser } from './level15-parser'
+import { parser as level16Parser } from './level16-parser'
+import { parser as level17Parser } from './level17-parser'
+import { parser as level18Parser} from './level18-parser'
+import { tags as t } from "@lezer/highlight";
+import { LRParser } from '@lezer/lr';
+
+interface languageSupport {
+    parser: LRParser,
+    styleTags: Record<string, any>
```

Comment:
Should the any be some sort of `Token` type?

Reply:
True!